cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -332,7 +332,10 @@ Usage: cardano-cli shelley governance action create-protocol-parameters-update -
 
   Create a protocol parameters update.
 
-Usage: cardano-cli shelley governance drep registration-certificate
+Usage: cardano-cli shelley governance drep 
+                                             ( registration-certificate
+                                             | metadata-hash
+                                             )
```

Comment:
The `governance drep metadata-hash` command doesn't make sense for eras before Conway.

Doing this should resolve that issue: https://github.com/input-output-hk/cardano-cli/pull/308/files#r1337080474